docs/storageclasses.md

```diff
@@ -1,34 +1,133 @@
+| Reclaim Policy <br> _(Supports Retain & Delete reclaim policy)_ </br> | Supported  | Yes <br> _(Test coverage exist for Delete reclaim policy)_ </br> |
+| [VolumeBindingMode](#volumebindingmode-optional) <br> _(Supports  Immediate & WaitForFirstConsumer modes)_ | Supported | Yes |
+| [allowedTopologies](#storageclass-with-custom-node-labels) | Supported | [Yes](https://github.com/openebs/lvm-localpv/tree/master/e2e-tests/experiments/functional/lvmpv-custom-topology#readme) |
+
+## StorageClass Options
+
+### AllowVolumeExpansion (Optional)
 
+Users can expand the volumes when the underlying StorageClass `allowVolumeExpansion` field set to true.
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-lvm
+allowVolumeExpansion: true     ## If set to true then dynamically it allows expansion of volume
+provisioner: local.csi.openebs.io
+parameters:
+  storage: "lvm"
+  vgpattern: "lvmvg.*"
 ```
-volgroup: "lvmvg"
+
+### MountOptions (Optional)
+
+Volumes that are provisioned via lvm-localpv will use the mount options specified in storageclass during volume mounting time.
+**Note**: Mount options are not validated. If mount options are invalid, then volume mount fails.
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-lvm
+provisioner: local.csi.openebs.io
+parameters:
+  storage: "lvm"
+  vgpattern: "lvmvg.*"
+  mountOptions:    ## Various mount options of volume can be specified here
+    - debug
 ```
 
-### vgpattern (*must* parameter if volgroup is not provided, otherwise optional)
+### Parameters
 
-vgpattern specifies the regular expression for the volume groups on node from which the volumes can be created. The *vgpattern* is the must argument if `volgroup` parameter is not provided in the storageclass. Here, in this case the driver will pick the volume groups matching the vgpattern with enough free capacity to accomodate the volume and will use the one which has largest capacity available for provisioning the volume.
+LVM-LocalPV storageclass supports various parameters for different use cases. Following are the supported parameters
+
+#### FsType (Optional)
```

Comment:
it is not clear whether VolumeBindingMode is part of Paramaters. Can we add themas bullet (or maybe number them).
```
#### - FsType (Optional)
```

Reply:
Pushed as bullet format under `Parameter` section